nmrfx-processor/src/main/java/org/nmrfx/processor/math/apache/ComplexSingularValueDecomposition.java

```diff
@@ -783,146 +777,4 @@ public static void aph(Complex[][] A, Rot P, int i1, int i2, int j1,
         }
     }
 
-    public static void main(String[] Args) {
-        File file = new File("/home/michael/csvd.txt");
```

Comment:
Removed this main method as it relied on a hardcoded path on someone's computer and it seems like its purpose was some kind of run speed benchmarking